wp-content/themes/core/components/header/subheader/css/subheader.pcss

```diff
@@ -0,0 +1,58 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Subheader
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-subheader {
+	position: relative;
+	display: flex;
+	align-items: center;
+	background: var(--color-grey-light);
+	padding:  var(--spacer-20) 0;
+	margin-bottom: var(--spacer-80);
+	min-height: 244px;
+
+	@media (--viewport-medium) {
+		min-height: 260px;
+	}
+
+	/* CASE: Page with featured image */
+	&.c-subheader--has-image {	
+		min-height: 308px;
+
+		@media (--viewport-medium) {
+			align-items: flex-end;
+			padding: var(--spacer-20) 0 var(--spacer-80) 0; 
+			min-height: 440px;
+		}
+	}
```

Comment:
You can reduce nesting here and place this at the top level as well, just below your `.c-subheader` selector 😃